.github/workflows/ci.yml

```diff
@@ -107,7 +107,7 @@ jobs:
         brew install zstd || brew upgrade zstd
         brew install lz4 || brew upgrade lz4
         brew install openssl@1.1 || brew upgrade openssl@1.1
-        #brew install homebrew/cask/osxfuse || brew upgrade homebrew/cask/osxfuse  # Required for Python llfuse module
+        brew install --cask macfuse || brew upgrade --cask macfuse  # Required for Python llfuse module
```

Comment:
I do not fully understand this patch. Why did you switch from osxfuse to macfuse?

Reply:
strictly taken, the switch was not required, it also worked with osxfuse after the macos downgrade.

but considering a later potential upgrade of macos to some more recent version again (after the hanging issue is analyzed and fixed), i wanted to switch to the more recent macfuse code to be prepared for that.